modules/lib/lib-repo/src/main/resources/lib/xp/examples/repo/modify.js

```diff
@@ -0,0 +1,26 @@
+/* global require, Java, testInstance*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['require' is already defined as a built-in global variable.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5056076)